README.md

```diff
@@ -183,6 +183,80 @@ By making a contribution to this project, I certify that:
+        |                | |  Native Abstractions |  |
+        |                | |      for Node.js     |  |
+        |                | +----------------------+  |
+        |                |     |             |       |
+   +----------------+    |     |             |       /
+   |  Node.js Core  |    |     |             |      /
+   |  Library API   |    |     |             |     /
+   +----------------+    |     |             |    /
+   |     js impl    |    |     |             |   /
+   +----------------+    |     |             |  /
+            |            |     |             | /
+   +--------------------------------------+  |/
+   | Node.js Application Binary Interface |  |
+   +--------------------------------------|  |
+   |              C/C++ impl              |  |
+   +--------------------------------------+  |
+                      |                      |
+        +---------------------------------------+
+        | Dependencies: v8, libuv, openssl, etc |
+        +---------------------------------------+
+```
+
+Node.js currently builds on top of several key dependencies including the V8 Javascript engine, libuv, openssl and others. The Node.js Application Binary Interface provides critical functionality such as the Event Loop which critical to how Node.js operates. The Node.js Core Library is the primary interface through which most Modules and Applications built on top of Node perform I/O operations, manipulate data, access the network, etc. Some modules and applications, however, go beyond the Core Library and bind directly to the Application Binary Interface and dependencies to perform more advanced operations. The Native Abstractions for Node.js (`nan`) is binary abstraction layer used to buffer module and application developers from changes in the Application Binary Interface and Dependencies.
+
+Due to the existing layering, modules and applications are sensitive not only to changes in the Core Library API, but the Application Binary Interface and dependendencies as well.
+
+Any API *addition* to either the Node.js Core Library API or Application Binary Interface must result in a *semver-minor* version increase.
+
+Any *backwards incompatible* change to either the Node.js Core Library API or Application Binary Interface must result in a *semver-major* version increase.
+
+Issue: Should any modification to the ABI or Dependencies that requires module or application developers to recompile force a *semver-major* change?
+
+APIs and default behaviors in the Node.js Core Library, Application Binary Interface, Dependencies and nan must not change within LTS Releases unless the change is required to address a critical security update.  
+
+Note that default or assumed behaviors and values are exposed via the API, ABI or Dependencies are considered to be part of the API. A change in a default value, for instance, counts as an API change as modules and applications may be written to assume certain defaults and could be broken.
```

Comment:
Perhaps: "Default or assumed behaviors that, if modified, could cause modules or applications to break are considered part of the API." This would exclude things like error messages which are *generally* informational but would catch such things as, say, openssl default cipher suites, etc.